packages/validate/test/validators.test.js

```diff
@@ -25,11 +25,13 @@ import {
   isDate,
   minDate,
   maxDate,
+  disabledDates,
```

Comment:
honestly I feel like smth is wrong with this name

it is not that it should be `disable` instead of `disabled`, but I mean it just does not fit into the current naming conventions

I think it should be smth into the direction of `isDateDisabled`

Reply:
I would be fine with both I guess. Any motivation for this name?